optuna/samplers/nsgaii/_sampler.py

```diff
@@ -104,6 +103,14 @@ class NSGAIISampler(BaseSampler):
                 versions without prior notice. See
                 https://github.com/optuna/optuna/releases/tag/v2.5.0.
 
+        elite_population_selection_strategy:
+            A process to select individuals to survive from current population pool. Defaults to
+            :class:`~optuna.samplers.nsgaii._elite_population_selection_strategy.NSGAIIElitePopulationSelectionStrategy`.
```

Comment:
The term "process" is also used in the context of `multiprocessing` and `subprocess`, which could potentially lead to confusion. Could you rephrase the explanation, please?

Reply:
Remove the word "process" from the docstring. Please take another look to confirm if the new one make sense.

Reply:
Thanks!